common/include/Utilities/FolderUtils.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+#include <fstream>
+#include <iostream>
+#include <iomanip>
+#include "FixedPointTypes.inl"
+#include "ghc/filesystem.h"
+#include "Path.h"
+
+namespace fs = ghc::filesystem;
+
+
+namespace Path
```

Comment:
Being as this seems to be a completely new namespace for future use, I'd recommend cutting out the fat that isn't needed anymore.  A lot of the functions inside are there because the old wx code relied on wxStrings heavily and such.  Going forward, if something is operating on a path/file, it should be using an `fs::path` to begin with, which removes the need for a lot of these functions.

I'm also not crazy about the non-descript namespace name of `Path`, I'd recommend leaving the old namespace as still `Path`, and make this one something more descriptive like `FileUtils`.  That will also have the side effect of minimizing the amount of `Path` -> `wxPath` changes you had to make here.